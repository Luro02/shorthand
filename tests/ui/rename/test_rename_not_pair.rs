use shorthand::ShortHand;

#[derive(ShortHand)]
struct Command {
    #[shorthand(rename(format))]
    #[shorthand(rename(format(xyz)))]
    first: usize,
}

fn main() {}
