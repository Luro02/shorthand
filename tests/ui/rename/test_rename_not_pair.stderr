error: Unexpected meta-item format `Path`
 --> $DIR/test_rename_not_pair.rs:5:24
  |
5 |     #[shorthand(rename(format))]
  |                        ^^^^^^

error: Unexpected meta-item format `List`
 --> $DIR/test_rename_not_pair.rs:6:24
  |
6 |     #[shorthand(rename(format(xyz)))]
  |                        ^^^^^^^^^^^
